src/scribe_data/utils.py

```diff
@@ -736,3 +737,70 @@ def check_index_exists(index_path: Path, overwrite_all: bool = False) -> bool:
         return choice == "Skip process"
 
     return False
+
+
+def check_qid_is_language(qid: str):
+    """
+    Parameters
+    ----------
+    qid : str
+        The QID to check Wikidata to see if it's a language and return its English label.
+
+    Outputs
+    -------
+    str
+        The English label of the Wikidata language entity.
+
+    Raises
+    ------
+    ValueError
+        An invalid QID that's not a language has been passed.
+    """
+    api_endpoint = "https://www.wikidata.org/w/rest.php/wikibase/v0"
+    request_string = f"{api_endpoint}/entities/items/{qid}"
+
+    request = requests.get(request_string, timeout=5)
+    request_result = request.json()
+
+    if request_result["statements"]["P31"]:
+        instance_of_values = request_result["statements"]["P31"]
+        for val in instance_of_values:
+            if val["value"]["content"] == "Q34770":
```

Comment:
:thinking: Coming across this.. 

was actually wondering if we do have it documented somewhere what `P31` or `Q34770` refer to in Wikidata (and also all other important IDs that we reference in code too). 

If not, should we? Was thinking that a quick markdown table could suffice really.

Reply:
Or actually - scratch that

It might be more helpful perhaps if that information is closer to where it's used in code. 
So in-source comments instead?

Reply:
Does another metadata file make sense for this? That way we reference an object and get the QID from a human readable object key?

Reply:
> Does another metadata file make sense for this?

hmm could be :thinking: 

Were you thinking of something like this?

```python
        instance_of_property = wikidata["property"]["instance-of"]
        instance_of_values = request_result["statements"][instance_of_property]
        for val in instance_of_values:
            if val["value"]["content"] == wikidata["entity"]["language"]:
```

Reply:
Along the lines of this, yes :)